resources/validate_assets.py

```diff
@@ -0,0 +1,102 @@
+from glob import glob
+import json
+from mcresources import utils
+
+assets_path = './src/main/resources/assets/'
+texture_forgiveness_paths = ('_fluff', 'block/burlap', 'block/molten_flow', 'yellow_bell', 'red_bell', 'green_bell', 'metal/full', 'plant/', 'sandstone/side', 'quiver', 'placed_item')
+
+def main():
+    model_locations = glob(assets_path + 'tfc/models/**/*.json', recursive=True)
+    state_locations = glob(assets_path + 'tfc/blockstates/**/*.json', recursive=True)
+    validate_model_parents(model_locations)
+    validate_textures(model_locations)
+    validate_blockstate_models(state_locations)
```

Comment:
In order to use this in validation, all that should be necessary is for these to accumulate errors, and throw a single `AssertionError` if any errors were printed.